pkg/config/config.go

```diff
@@ -253,14 +268,17 @@ func (v *validatorConfig) check(name validatorName, requiredInputs []string) err
 	return nil
 }
 
+// ModuleID is a unique identifier for a module in a blueprint
+type ModuleID string
```

Comment:
nit: Should we validate this string is not empty?

Reply:
We should, but it's out of scope